[2025-03-31T08:52:52.933+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-31T08:52:52.950+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: forex_ml_pipeline.create_s3_bucket manual__2025-03-31T08:52:50.974598+00:00 [queued]>
[2025-03-31T08:52:52.962+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: forex_ml_pipeline.create_s3_bucket manual__2025-03-31T08:52:50.974598+00:00 [queued]>
[2025-03-31T08:52:52.963+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 4
[2025-03-31T08:52:52.984+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): create_s3_bucket> on 2025-03-31 08:52:50.974598+00:00
[2025-03-31T08:52:52.997+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'forex_ml_pipeline', 'create_s3_bucket', 'manual__2025-03-31T08:52:50.974598+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/forex_ml_pipeline.py', '--cfg-path', '/tmp/tmpw32srq_6']
[2025-03-31T08:52:53.001+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask create_s3_bucket
[2025-03-31T08:52:53.001+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=337) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-31T08:52:53.005+0000] {standard_task_runner.py:72} INFO - Started process 341 to run task
[2025-03-31T08:52:53.075+0000] {task_command.py:467} INFO - Running <TaskInstance: forex_ml_pipeline.create_s3_bucket manual__2025-03-31T08:52:50.974598+00:00 [running]> on host 1f5073dcf149
[2025-03-31T08:52:53.173+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mikhail' AIRFLOW_CTX_DAG_ID='forex_ml_pipeline' AIRFLOW_CTX_TASK_ID='create_s3_bucket' AIRFLOW_CTX_EXECUTION_DATE='2025-03-31T08:52:50.974598+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-31T08:52:50.974598+00:00'
[2025-03-31T08:52:53.175+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-31T08:52:53.201+0000] {base.py:84} INFO - Retrieving connection 'my_s3_conn'
[2025-03-31T08:52:53.202+0000] {connection_wrapper.py:328} INFO - AWS Connection (conn_id='my_s3_conn', conn_type='aws') credentials retrieved from extra.
[2025-03-31T08:52:55.641+0000] {logging_mixin.py:190} INFO - Bucket 'usd-php-ml-pipeline-bucket' already exists. Skipping creation.
[2025-03-31T08:52:55.642+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-31T08:52:55.671+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-31T08:52:55.672+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=forex_ml_pipeline, task_id=create_s3_bucket, run_id=manual__2025-03-31T08:52:50.974598+00:00, execution_date=20250331T085250, start_date=20250331T085252, end_date=20250331T085255
[2025-03-31T08:52:55.713+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-31T08:52:55.744+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-31T08:52:55.749+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
